src/expected_loglik.jl

```diff
@@ -73,9 +73,20 @@ function expected_loglik(
     # (see e.g. en.wikipedia.org/wiki/Gauss%E2%80%93Hermite_quadrature)
     xs, ws = gausshermite(gh.n_points)
     # size(fs): (length(y), n_points)
-    fs = √2 * std.(q_f) .* xs' .+ mean.(q_f)
-    lls = loglikelihood.(lik.(fs), y)
-    return sum((1 / √π) * lls * ws)
+    return sum(
+        Broadcast.instantiate(
+            Broadcast.broadcasted(q_f, y) do q, y
```

Comment:
It'd be nice to not overload the same name, just to be clear... how about changing the function signature to `ys` and `q_fs`?